features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/Comments/CommentAdd.jsx

```diff
@@ -71,13 +75,14 @@ class CommentAdd extends React.Component {
         super(props);
         this.state = {
             content: '',
+            replyIndex: -1,
```

Comment:
Do we need a reply index when adding a comment? Shouldn't it the parent comment ID ?